docs/src/multi-physics_coupling.md

```diff
@@ -0,0 +1,37 @@
+# [Multi-physics coupling](@id multi-physics-coupling)
+A complex simulation can consist of different spatial domains in which
+different equations are being solved, different numerical methods being used
+or the grid structure is different.
+One example would be a fluid in a tank and an extended hot plate attached to it.
+We would then like to solve the Navier-Stokes equations in the fluid domain
+and the heat conduction equations in the plate.
+The coupling would happen at the interface through the exchange of thermal energy.
+
+
+## Converter Coupling
+We can have the case where the two systems do not share any variables, but
+share some of the physics.
+Here, the same physics is just represented in a different form and with
+different variables.
+This is the case for a fluid system on one side and a Vlasov system on the other.
+To translate the fields from one description to the other one needs to use
+converter functions.
+
+In the general case, we have a system A with $m$ variables $u_{A,i}$ and another
```

Comment:
```suggestion
In the general case, we have a system A with ``m`` variables ``u_{A,i}`` and another
```
According to the [docs of Documenter.jl](https://documenter.juliadocs.org/stable/man/latex/#Inline-Equations), I suggest to use two backticks instead of `$` to switch into math mode as we do elsewhere.

Reply:
Thanks!